docs/app/layout.tsx

```diff
@@ -62,8 +62,9 @@ const navbar = (
       <NextraLogo
         height="20"
         className={cn(
+          'transition-all duration-1000',
```

Comment:
```suggestion
          'hover:transition-all duration-1000',
```
now transition is different with current behaviour, we should apply transition only on `:hover` state